content/21.nutraceuticals.md

```diff
@@ -306,11 +309,11 @@ Therefore, the public response once again seems to have adopted supplements prom
 In this review, we report the findings to date of analyses of several dietary supplements and nutraceuticals.
 While existing evidence suggests potential benefits of n-3 PUFA and probiotic supplementation for COVID-19 treatment and prophylaxis, clinical data is still lacking, although trials are underway.
 Both zinc and vitamin C supplementation in hospitalized patients seem to be associated with positive outcomes; however, further clinical trials are required.
-In any case, both vitamin C and zinc intake are part of a healthy diet and both likely presents minimal risk when supplemented for, though their potential prophylactic or therapeutic effects against COVID-19 are yet to be determined.
+In any case, vitamin C and zinc intake are part of a healthy diet and likely present minimal risk when supplemented, though their potential prophylactic or therapeutic effects against COVID-19 is yet to be determined.
```

Comment:
```suggestion
In any case, vitamin C and zinc intake are part of a healthy diet and likely present minimal risk when supplemented, though their potential prophylactic or therapeutic effects against COVID-19 are yet to be determined.
```

Reply:
This was the reviewer suggestion to change are -> is ... I changed one because I wasn't sure myself... I thought using are was fine

Reply:
I actually think I understand what they're saying now -- "or" makes it weird. Probably the most-correct would be either "any potential prophylactic..." or alternatively "their potential prophylactic and therapeutic effects..." and then use "are", but I doubt it's worth getting too worried about!